todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,48 @@
+from View_Model import ViewModel
+from todo_app.data.item import Item
+
+test_items = [
+        Item(1, "New Todo", "To Do"),
+        Item(2, "Completed Todo", "Complete"),
+        Item(3, "Active Todo", "Active"),
+        Item(4, "Active 2 Todo", "Active")
+    ]
+
+def test_view_model_complete_property_only_provides_completed_items_and_nothing_else():
+    # Arrange
+    ## Pre work to do
+    view_model = ViewModel(test_items)
+    
+    # Act
+    ## filter items for complete.
+    complete_items = view_model.complete_items
+
+    # Assert
+    ## Check the filtering for an expected quantity
+    assert len(complete_items) == 1   # 1 complete in test list
```

Comment:
This makes me pretty confident that we've got the right filtering, but a nice addition might be to check that you do get the right items back, e.g. something like:
```suggestion
    assert len(complete_items) == 1   # 1 complete in test list
    for item in complete_items:
        assert item.status == "Complete"
```